benchmarks/.gitignore

```diff
@@ -1,4 +1,3 @@
-**/*.bench.json
-bench.json
+*.json
```

Comment:
Not sure about this—some JSON files are actually meant to be checked in. For example, the outputs of the parser tests:
https://github.com/sampsyo/bril/blob/main/test/parse/add.json

Reply:
Oops, thanks, didn't realize this was in a subdirectory!